WordPress/src/main/res/values/strings.xml

```diff
@@ -1676,6 +1676,10 @@
     <string name="photo_picker_choose_video">Choose video from device</string>
     <string name="photo_picker_capture_photo">Take photo</string>
     <string name="photo_picker_capture_video">Take video</string>
+    <string name="photo_picker_soft_ask_label">%s needs permissions to access your photos</string>
+    <string name="photo_picker_soft_ask_allow">Allow</string>
+    <string name="photo_picker_soft_ask_permissions_denied">%s was denied permissions to access your photos</string>
```

Comment:
> I went back and forth about this. I figured most people are adding pictures so using "photos" made the most sense, but it isn't really accurate since it shows videos as well. What do you think?

Both ways are fine to me. Just noticed that we've used the wording `optimize images` in settings screen, in section about compressing/resizing images. Worth to change both to `photos`? 

Also noticed that the system dialog, shown when I clicked on Allow, has the following wording `Allow <app-name> to access photos, media, and files on your devices`.